README.md

```diff
@@ -20,7 +20,7 @@ To install this website on your computer, first [install grav core](https://getg
 git clone https://github.com/getgrav/grav.git userfrosting-learn
 cd userfrosting-learn
 rm -r user
-git clone https://github.com/userfrosting/learn.git user
+git clone --recursive https://github.com/userfrosting/learn.git user
```

Comment:
Oh, that recursion. 

Then yes, it's possible. I guess the `--recursive` doesn't matter in the end, since the setup also adds `git submodule update --remote --merge` step.

Plus :
> Master is still in `user/pages/`. Once this is merged, the base site could be separated from the "master" documentation in a new branch.

I was thinking of a `website` branch, with the base site and the submodules, then removing **from each branch** (4.0, 4.1, develop, hotfix & master) the unnecessary files, keeping only `/pages` and `/config`